lib/universal/mapper.js

```diff
@@ -579,6 +579,8 @@ function createCommonGAForm(facade, settings) {
   var app = facade.proxy('context.app') || {};
   var traits = facade.traits();
   var options = facade.options('Google Analytics');
+  var os = facade.proxy('context.os');
```

Comment:
It's fine to send `context.os` without `name` (and same for the other nested values). Just because someone provided `context.os` doesn't mean they provided all the nested values too. 

We should proxy `context.os.name`, `context.device.manufacturer` and `context.os.version` and check if they are defined or not.